pycalphad/tests/test_model.py

```diff
@@ -35,7 +35,7 @@ def test_model_ne():
 
 def test_export_import():
     "Equivalence of Model using re-imported database."
-    test_model = Model(Database.from_string(ALNIPT_DBF.to_string(fmt='tdb'), fmt='tdb'), ['PT', 'NI', 'VA'], 'FCC_L12')
+    test_model = Model(Database.from_string(ALNIPT_DBF.to_string(fmt='tdb', if_incompatible='ignore'), fmt='tdb'), ['PT', 'NI', 'VA'], 'FCC_L12')
```

Comment:
`if_incompatible=ignore` is fine here since the warning is expected and safe to ignore; I'm just wondering about the default setting. If someone has their own TDB cleaning/whatever script for personal use and it happens to deal with TDBs with long symbol names, then updating to the new release of pycalphad will break their workflow until they read the documentation. A default setting of `if_incompatible=warn` for a few releases would give them an opportunity to react without the abrupt change.